tests/sentry/api/endpoints/test_release_threshold_details.py

```diff
@@ -0,0 +1,73 @@
+from django.urls import reverse
+
+from sentry.models.environment import Environment
+from sentry.models.release_threshold.releasethreshold import ReleaseThreshold
+from sentry.testutils.cases import APITestCase
+
+
+class ReleaseThresholdDetailsTest(APITestCase):
```

Comment:
nit: switch to using `get_success_response` and `get_error_response` over `client.get`.
```suggestion
class ReleaseThresholdDetailsTest(APITestCase):
    endpoint = "sentry-api-0-project-release-thresholds-details"
    method = "get"
```